utils/change_ini_param.py

```diff
@@ -25,33 +27,34 @@
 
 
 def main():
-    parser = OptionParser(usage='usage: %prog [options] option value')
-    parser.add_option('--config-file', default=MY_CNF,
+    """Main module function."""
+    parser = ArgumentParser(usage='%(prog)s [options] option value')
+    parser.add_argument('--config-file', default=MY_CNF,
                       help='Path to configuration file', metavar='CONFIG_FILE')
-    parser.add_option('--section', default=SECTION,
+    parser.add_argument('--section', default=SECTION,
                       help='Section name which the specified option '
                            'belongs to',
                       metavar='SECTION')
-    options, args = parser.parse_args()
+    args = parser.parse_args()
 
-    if len(args) != 2:
+    if len(vars(args)) != 2:
```

Comment:
Maybe we need to remove this condition?
Because the `config_file` and` section` arguments have default values. And situation when `len (args)` is not equal to 2 will never happen.
@eugulixes what do you think about it?